src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java

```diff
@@ -0,0 +1,84 @@
+package com.iridium.iridiumskyblock.commands;
+
+import com.iridium.iridiumcore.utils.StringUtils;
+import com.iridium.iridiumskyblock.IridiumSkyblock;
+import com.iridium.iridiumskyblock.IslandRank;
+import com.iridium.iridiumskyblock.database.Island;
+import com.iridium.iridiumskyblock.database.User;
+import org.bukkit.Bukkit;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+public class RenameCommand extends Command {
+
+    public RenameCommand() {
+        super(Collections.singletonList("rename"), "Change your the island name", "", true);
```

Comment:
"Change the name of your island"

Reply:
Oh, it sounded really bad but I think it's also correct. It's up to you

Reply:
Hes doesnt make sence, use the one das sent

Reply:
or make it `Change your island name`